packages/visx-wordcloud/src/useWordcloud.ts

```diff
@@ -0,0 +1,132 @@
+import { useEffect, useState } from 'react';
+import d3Cloud from 'd3-cloud';
+
+export interface BaseDatum {
+  text: string;
+}
+
+export interface WordcloudConfig<Datum extends BaseDatum> {
+  /**
+   * Width of the wordcloud layout.
+   *
+   * @default 0
+   */
+  width?: number;
+  /**
+   * Height of the wordcloud layout.
+   *
+   * @default 0
+   */
+  height?: number;
+  /**
+   * Sets the words array.
+   *
+   * @default []
+   */
+  words?: Datum[];
```

Comment:
Should `undefined` be allowed for this field?

Reply:
Yeah having this field `undefined` makes no sense, I just kept the API parity with `d3-cloud`. I'll make this required, and on the same note, I think the width and height should be required as well, WDYT?

Reply:
requiring `width/height` makes sense to me since you'd get no output otherwise. some lib consumers aren't familiar with the `d3` intricacies and have been confused by things like that in the past)